bus-mapping/src/rpc.rs

```diff
@@ -0,0 +1,58 @@
+//! Module which contains all the RPC calls that are needed at any point to query a Geth node in order to get a Block, Tx or Trace info.
+
+use crate::eth_types::{Block, Hash, Transaction};
+use crate::Error;
+use ethers_providers::JsonRpcClient;
+
+/// Serialize a type.
+///
+/// # Panics
+///
+/// If the type returns an error during serialization.
+pub fn serialize<T: serde::Serialize>(t: &T) -> serde_json::Value {
+    serde_json::to_value(t).expect("Types never fail to serialize.")
+}
+
+/// Placeholder structure designed to contain the methods that the BusMapping needs in order to enable Geth queries.
+pub struct GethQueries<P: JsonRpcClient>(P);
```

Comment:
Suggestion for naming: Maybe call it `GethClient`?

Reply:
I like it! Let's add it!!